app/src/main/java/com/framgia/quangtran/music_42/mediaplayer/MediaPlayerManager.java

```diff
@@ -0,0 +1,183 @@
+package com.framgia.quangtran.music_42.mediaplayer;
+
+import android.content.Context;
+import android.media.MediaPlayer;
+import android.net.Uri;
+
+import com.framgia.quangtran.music_42.data.model.Track;
+
+import java.io.IOException;
+import java.util.List;
+
+public class MediaPlayerManager extends MediaPlayerSetting implements MediaPlayerInterface
```

Comment:
```suggestion
public class TracksPlayerManager extends MediaPlayerSetting implements MediaPlayerInterface
```
Em shift + f6 để rename thành TracksPlayerManager thì nó đúng hơn.